src/Symfony/Component/Messenger/Transport/Receiver/ForceConsumptionReceiver.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Messenger\Transport\Receiver;
+
+use Psr\Log\LoggerInterface;
+
+/**
+ * @author Mathias STRASSER <contact@roukmoute.fr>
+ *
+ * @experimental in 4.2
+ */
+final class ForceConsumptionReceiver implements ReceiverInterface
+{
+    /** @var ReceiverInterface */
+    private $decoratedReceiver;
+
+    /** @var LoggerInterface */
```

Comment:
symfony does not add phpdocs for private properties as it can be inferred from the constructor. also i would need to be `LoggerInterface|null`